mm2src/coins/utxo/bch.rs

```diff
@@ -1263,6 +1264,15 @@ impl MmCoin for BchCoin {
     fn is_coin_protocol_supported(&self, info: &Option<Vec<u8>>) -> bool {
         utxo_common::is_coin_protocol_supported(self, info)
     }
+
+    fn on_disabled(&self) -> Result<(), AbortedError> { AbortableSystem::abort_all(&self.as_ref().abortable_system) }
+
+    fn on_token_deactivated(&self, ticker: &str) -> Result<(), String> {
+        if let Ok(tokens) = self.slp_tokens_infos.try_lock().as_deref_mut() {
```

Comment:
Consider using `lock` instead of `try_lock`.
Also I think `on_token_deactivated` can return nothing instead of the result